NitroxLauncher/ServerPage.xaml

```diff
@@ -307,36 +307,28 @@
+                        <ColumnDefinition Width="300" />
                     </Grid.ColumnDefinitions>
                     <StackPanel Grid.Column="0" VerticalAlignment="Center">
-                        <TextBlock FontFamily="{StaticResource Roboto}" Foreground="White" FontSize="12"><Run Text="GAME VERSION"/></TextBlock>
-                        <Border Height="43" Width="269" Margin="0,4,0,0" BorderBrush="#33FFFFFF" BorderThickness="1" CornerRadius="4">
-                            <ComboBox x:Name="CBox" Background="#9928292C" Foreground="White" FontSize="16" Padding="12,12,0,0" SelectedIndex="0" SelectionChanged="OnSelectionChange">
-                                <ComboBox.Resources>
-                                    <SolidColorBrush x:Key="{x:Static SystemColors.WindowBrushKey}" Color="#9928292C" />
-                                </ComboBox.Resources>
-
-                                <ComboBox.FontFamily>
-                                    <StaticResource ResourceKey="Roboto"/>
-                                </ComboBox.FontFamily>
-
-                                <ComboBoxItem FontFamily="{StaticResource Roboto}" x:Name="CBWindow" Foreground="White" HorizontalAlignment="Stretch" FontSize="16" Content="Command line window" ToolTip="Server will run in a seperate window (CMD)" Tag="external"/>
-                                <ComboBoxItem FontFamily="{StaticResource Roboto}" x:Name="CBInner" Foreground="White" HorizontalAlignment="Stretch" FontSize="16" Content="Launcher window" ToolTip="Server will run in the launcher." Tag="embedded" />
-                            </ComboBox>
-                            <!--<TextBlock Foreground="White" HorizontalAlignment="Left" FontSize="16" Margin="12,12,0,0">Only base game available</TextBlock>-->
-                        </Border>
+                        <TextBlock FontFamily="{StaticResource Roboto}" Foreground="White" FontSize="22"><Run Text="SERVER TYPE"/></TextBlock>
+                        <RadioButton Name="RBIsDocked" GroupName="ServerType" Content="Docked console" Margin="5" FontFamily="{StaticResource Roboto}" Foreground="LightGray" FontSize="14" HorizontalAlignment="Left" VerticalAlignment="Top" Checked="RBServer_Checked"/>
+                        <RadioButton Name="RBIsExternal" GroupName="ServerType" Content="External command prompt" Margin="5" FontFamily="{StaticResource Roboto}" Foreground="LightGray" FontSize="14" HorizontalAlignment="Left" VerticalAlignment="Top" Checked="RBServer_Checked"/>
+                        <!--
+                        <CheckBox Name="ChkIsExternal" Content="External command prompt process" Margin="0,5,0,5" FontFamily="{StaticResource Roboto}" Foreground="LightGray" FontSize="16"/>
+                        <TextBlock FontFamily="{StaticResource Roboto}" Foreground="LightGray" FontSize="14" TextWrapping="Wrap"><Run Text="Launches server in external command prompt process."/></TextBlock>
+                        -->
```

Comment:
Let's remove dead code.

Reply:
Whoops. Got it. Related to dead code...although I'm brand new to WPF, there is a huge block in the "Page Resources" tag that I think is now OBE since the ComboBox is gone...I'll look into that next.